docs/snippets/all/howto/check_connection_status.rs

```diff
@@ -0,0 +1,40 @@
+//! Continuously print the connection status of a pending gRPC connection.
+//!
+//! This feature is experimental and may change in future releases.
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let rec = rerun::RecordingStreamBuilder::new("rerun_example_check_connection_status")
+        .connect_grpc()?;
+
+    let (tx, rx) = std::sync::mpsc::channel();
+
+    loop {
+        let tx = tx.clone();
+        rec.inspect_sink(move |sink| {
```

Comment:
As they say, nothing more permanent than a temporary solution 😄

Reply:
admittedly that part makes me mega nervous. This solution is _definitely_ not great and I'd hope it will fold into something much nicer in the future, but maybe I just made the nicer future harder still? :/

Reply:
:shrug: this seems easy to rip out, so I'm not too worried